client/components/TestReview/index.jsx

```diff
@@ -0,0 +1,270 @@
+                    <Link
+                        to={`/data-management/${testPlanVersion.testPlan.directory}`}
+                    >
+                        (View all versions of this test plan)
+                    </Link>
+                </li>
+                <li>
+                    <strong>Version History:&nbsp;</strong>
+                    <Ul>
+                        <li>
+                            {`R&D completed on ${convertDateToString(
+                                testPlanVersion.updatedAt,
+                                'MMM D, YYYY'
+                            )}.`}
+                        </li>
+                        {!testPlanVersion.draftPhaseReachedAt ? null : (
+                            <li>
+                                {`ARIA-AT draft review process started on ` +
+                                    `${convertDateToString(
+                                        testPlanVersion.draftPhaseReachedAt,
+                                        'MMM D, YYYY'
+                                    )} ` +
+                                    `for this version.`}
+                            </li>
+                        )}
+                        {!testPlanVersion.candidatePhaseReachedAt ? null : (
+                            <li>
+                                {`ARIA-AT candidate review process started on ` +
+                                    `${convertDateToString(
+                                        testPlanVersion.candidatePhaseReachedAt,
+                                        'MMM D, YYYY'
+                                    )} ` +
+                                    `for this version.`}
+                            </li>
+                        )}
+                        {!testPlanVersion.recommendedPhaseReachedAt ? null : (
+                            <li>
+                                {`Version reached ARIA-AT recommended status on ` +
+                                    `${convertDateToString(
+                                        testPlanVersion.recommendedPhaseReachedAt,
+                                        'MMM D, YYYY'
+                                    )}.`}
+                            </li>
+                        )}
+                        {!testPlanVersion.deprecatedAt ? null : (
+                            <li>
+                                {`Version deprecated on ` +
+                                    `${convertDateToString(
+                                        testPlanVersion.deprecatedAt,
+                                        'MMM D, YYYY'
+                                    )}.`}
+                            </li>
+                        )}
+                    </Ul>
+                </li>
+                <li>
+                    <strong>Commit:&nbsp;</strong>
+                    {testPlanVersion.gitMessage}
+                </li>
+            </ul>
+            <h2>List of Tests</h2>
```

Comment:
This component is pretty lengthy. I feel like the filtered tests could become their own component fairly easy. This would shorten this component and it seems likely that it could be reused elsewhere at some point since this section has a lot of standalone use value.

Reply:
I think this is a change that probably makes more sense to implement when this content is needed somewhere else, because I think for now the connective tissue needed like prop types might undercut some of the space saving benefits, and the component interface might be negatively affected by the need to support filtering out irrelevant ATs.